cpp/pybind/t/geometry/pointcloud.cpp

```diff
@@ -253,6 +253,12 @@ The attributes of the point cloud have different levels::
                    "nb_points"_a, "search_radius"_a,
                    "Remove points that have less than nb_points neighbors in a "
                    "sphere of a given search radius.");
+    pointcloud.def(
+            "remove_statistical_outliers",
+            &PointCloud::RemoveStatisticalOutliers, "nb_neighbors"_a,
+            "std_ratio"_a,
+            "Remove points that have less than nb_points in a sphere of "
+            "a given radius. This function is not recommended to use on GPU.");
```

Comment:
Same, talk about what is being returned here.

Reply:
I notice that the other functions in pybind do not mention the return as well. Should we apply the same modification to them? @yxlao